presidio-analyzer/analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -30,7 +30,11 @@ def __init__(self, recognizer_store_api=RecognizerStoreApi(),
             self.recognizers = recognizers
         else:
             self.recognizers = []
+        # loaded_hash is used to store the currently loaded hash of in memory
+        # recognizers, to avoid fetching the recognizers from t
```

Comment:
Sentence is cut in the middle. A suggestion:
loaded_hash is the hash value of the recognizers in the recognizer store. It is used to avoid fetching recognizers when there hasn't been a change to the recognizers state.